tests/unit/phpunit/include/MVC/View/ViewFactoryTest.php

```diff
@@ -1,11 +1,13 @@
 <?php
 
 
-class ViewFactoryTest extends SuiteCRM\StateCheckerPHPUnitTestCaseAbstract
+use SuiteCRM\Test\SuitePHPUnitFrameworkTestCase;
+
+class ViewFactoryTest extends SuitePHPUnitFrameworkTestCase
 {
     public function testloadView()
     {
-        $state = new SuiteCRM\StateSaver();
+
         
```

Comment:
Remove all this extra space.

Reply:
Actually, let's just make a note to clean up the `tests/unit/phpunit/include/` directory in a PR later.